app/controllers/discourse_automation/automations_controller.rb

```diff
@@ -2,12 +2,27 @@
 
 module DiscourseAutomation
   class AutomationsController < ApplicationController
-    before_action :ensure_admin
+    before_action :ensure_admin, except: [:post_checked]
+    before_action :ensure_logged_in, only: [:post_checked]
 
     def trigger
       automation = DiscourseAutomation::Automation.find(params[:id])
       automation.trigger!(params.merge(kind: DiscourseAutomation::Triggerable::API_CALL))
       render json: success_json
     end
+
+    def post_checked
```

Comment:
Not sure if this is the right place for this action? Maybe we should create a controller for that. Like we did for the [user global notices](https://github.com/discourse/discourse-automation/blob/main/app/controllers/discourse_automation/user_global_notices_controller.rb)